driver/requirements.txt

```diff
@@ -3,3 +3,7 @@ pytest-subtests
 pytest-cov
 dacite
 mpi4py
+numpy
+netCDF4
+xarray
+zarr
```

Comment:
Do any of these need to be added to driver's `setup.py`?

Reply:
Yes, added